### ISSUE TITLE
pseudonyms should not be used in the LICENSE file

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#8762]
|[<img alt="glyph's avatar" src="https://avatars.githubusercontent.com/u/716529?s=50" width="50" height="50">](https://github.com/glyph)| @glyph reported|
|-|-|
|Trac ID|trac#8762|
|Type|task|
|Created|2016-08-16 04:11:23Z|

Recently a commit added the name "Charmander" to the LICENSE file.  In addition to this being a trademark of the Pokémon company and therefore potentially problematic for us legally, it also doesn't properly identify a legal person.  The vague fiction upon which we have built our project's license statement policy is that "Twisted Matrix Labs" is a shorthand (this is allowed) that refers to the authors; authors which retain copyright are listed in the LICENSE file.  Since it's a legal document it needs to use legal names.

Note that yours truly does not have a pseudonym in the LICENSE file either :-).

Charmander does not wish to disclose their legal name, and I would not ask them to do so; the point of a LICENSE file is not doxing.  However, it does make it problematic because the point of having legal persons listed there is to allow licensees to examine the list of copyright holders.  However, according to an Actual Lawyer™ they can release the copyright for their changes into the public domain and then don't need to be identified as a copyright holder.

<details><summary>Searchable metadata</summary>

```
trac-id__8762 8762
type__task task
reporter__glyph glyph
priority__highest highest
milestone__None None
branch__ 
branch_author__charmander charmander
status__closed closed
resolution__fixed fixed
component__core core
keywords__None None
time__1471320683822908 1471320683822908
changetime__1474197555542286 1474197555542286
version__None None
owner__Amber_Brown__HawkOwl___hawkowl_____ Amber Brown (HawkOwl) <hawkowl@...>

```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="charmander's avatar" src="https://avatars.githubusercontent.com/u/1889843?s=50" width="50" height="50">](https://github.com/charmander)<a name="note_1"></a>|@charmander commented|
|-|-|

https://github.com/twisted/twisted/pull/500

**Comment by twisted-trac** [comment]
|[<img alt="hawkowl's avatar" src="https://avatars.githubusercontent.com/u/3307100?s=50" width="50" height="50">](https://github.com/hawkowl)<a name="note_4"></a>|@hawkowl set owner to @hawkowl<br>@hawkowl set status to `closed`|
|-|-|

In changeset dc7488c02e7bc36208fbe7a8e22b7bd506247f56
```
#!CommitTicketReference repository="" revision="dc7488c02e7bc36208fbe7a8e22b7bd506247f56"
Merge charmander-trivial-change-license-entry-8762: Remove psuedonym from LICENSE

Author: charmander
Reviewer: hawkowl
Fixes: #8762

A pseudonym was removed from the license and the associated changes were made explicitly CC0.
```

**Comment by twisted-trac** [comment]
|[<img alt="hawkowl's avatar" src="https://avatars.githubusercontent.com/u/3307100?s=50" width="50" height="50">](https://github.com/hawkowl)<a name="note_5"></a>|@hawkowl commented|
|-|-|

[mass edit] Removing review from closed tickets.
